src/test/java/io/github/acm19/aws/interceptor/test/AmazonOpenSearchServiceSample.java

```diff
@@ -8,59 +8,70 @@
 import org.apache.http.client.entity.GzipCompressingEntity;
 import org.apache.http.client.methods.HttpGet;
 import org.apache.http.client.methods.HttpPost;
-import org.apache.http.entity.BasicHttpEntity;
+// import org.apache.http.entity.BasicHttpEntity;
```

Comment:
Remove

Reply:
There's commented code reproducing a bug below, would prefer to leave it.

Reply:
But this class is not in that code. I'm not sure I'm following.

Reply:
removed